java/src/jmri/InstanceManager.java

```diff
@@ -263,6 +316,40 @@
         return getDefault().getInstance(type);
     }
 
+    /**
+     * Retrieve the last object of type T that was registered with
+     * {@link #store(java.lang.Object, java.lang.Class) }.
+     *
+     * Intended for use with i.e. scripts where access to the class type is inconvenient.
+     * In Java code where typing is enforced, use {@link #getNullableDefault(Class)}.
+     * <p>
+     * Unless specifically set, the default is the last object stored, see the
+     * {@link #setDefault(java.lang.Class, java.lang.Object) } method.
+     * <p>
+     * In some cases, InstanceManager can create the object the first time it's
+     * requested. For more on that, see the class comment.
+     * <p>
+     * In most cases, system configuration assures the existence of a default
+     * object, but this method also handles the case where one doesn't exist.
+     * Use {@link #getDefault(java.lang.Class)} when the object is guaranteed to
+     * exist.
+     *
+     * @param className Fully qualified class name
+     * @return The default object for type.
+     * @see #getOptionalDefault(java.lang.Class)
+     */
+    @CheckForNull
+    public static Object getNullableDefault(@Nonnull String className) {
+        Class<?> type;
+        try {
+            type = Class.forName(className);
+        } catch (ClassNotFoundException ex) {
+            log.error("No class found: {}", className);
+            throw new IllegalArgumentException(ex);
```

Comment:
I think it would be good to add `@throws IllegalArgumentException` to the Javadoc for this method.
One possible interpretation of `getNullableDefault()` is that it would return null if the class name is wrong. By adding `@throws` to the Javadoc, it's clear what happens then a bad class name is given.